edi_sale_edifact_oca/demo/edi_exchange_type.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="demo_edi_exc_type_edifact_order_in" model="edi.exchange.type">
+      <field name="backend_id" ref="edi_edifact_oca.edi_backend_edifact_demo" />
+      <field name="backend_type_id" ref="edi_edifact_oca.edi_backend_type_edifact" />
+      <field name="name">Demo EDIFACT Sale Order</field>
+      <field name="code">demo_EDIFACT_SaleOrder_in</field>
+      <field name="direction">input</field>
+      <field name="exchange_file_ext">txt</field>
+      <field name="advanced_settings_edit">
+ components:
+    process:
+       usage: input.process.sale.order
+ sale_order_import:
```

Comment:
this will change according to my proposition on https://github.com/OCA/edi-framework/pull/31